src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -151,75 +156,140 @@ object SbtBundle extends AutoPlugin {
 
   override def trigger = AllRequirements
 
-  override def projectSettings = Seq(
-    compatibilityVersion := version.value.takeWhile(_ != '.'),
-    NativePackagerKeys.packageName in Bundle := normalizedName.value + "-v" + compatibilityVersion.value,
-
-    system := (normalizedName in Bundle).value,
-    systemVersion := compatibilityVersion.value,
-    roles := Set("web"),
-
-    bundleConf := getConfig.value,
-    bundleType := Universal,
-    startCommand := Seq(
-      (file((normalizedName in Bundle).value) / "bin" / (executableScriptName in Universal).value).getPath,
-      s"-J-Xms${memory.value.round1k.underlying}",
-      s"-J-Xmx${memory.value.round1k.underlying}"
-    ),
-    endpoints := Map("web" -> Endpoint("http", 0, Set(URI("http://:9000")))),
-    checkInitialDelay := 3.seconds,
-    checks := Seq.empty,
-    NativePackagerKeys.dist in Bundle := Def.taskDyn {
-      Def.task {
-        createDist(bundleType.value)
-      }.value
-    }.value,
-    NativePackagerKeys.stage in Bundle := Def.taskDyn {
-      Def.task {
-        stageBundle(bundleType.value)
-      }.value
-    }.value,
-    NativePackagerKeys.dist in BundleConfiguration := Def.taskDyn {
-      Def.task {
-        createConfiguration()
-      }.value
-    }.value,
-    NativePackagerKeys.stage in BundleConfiguration := Def.taskDyn {
-      Def.task {
-        stageConfiguration()
-      }.value
-    }.value,
-    NativePackagerKeys.stagingDirectory in Bundle := (target in Bundle).value / "stage",
-    NativePackagerKeys.stagingDirectory in BundleConfiguration := (target in BundleConfiguration).value / "stage",
-    target in Bundle := target.value / "bundle",
-    target in BundleConfiguration := target.value / "bundle-configuration",
-    sourceDirectory in BundleConfiguration := (sourceDirectory in Universal).value.getParentFile / "bundle-configuration",
-    configurationName := "default"
-  )
-
-  private def createDist(bundleTypeConfig: Configuration): Def.Initialize[Task[File]] = Def.task {
-    val bundleTarget = (target in Bundle).value
-    val configTarget = bundleTarget / "tmp"
+  override def projectSettings =
+    bundleSettings(Bundle) ++ configurationSettings(BundleConfiguration) ++
+      Seq(
+        bundleType := Universal,
+        checkInitialDelay := 3.seconds,
+        checks := Seq.empty,
+        compatibilityVersion := (version in Bundle).value.takeWhile(_ != '.'),
+        configurationName := "default",
+        endpoints := Map("web" -> Endpoint("http", 0, Set(URI("http://:9000")))),
```

Comment:
This has been the default since day one. :-) I figured then that most apps would be Play and http based - these are effectively Play defaults.